apps/astarte_housekeeping/lib/astarte_housekeeping/release_tasks.ex

```diff
@@ -84,7 +85,7 @@ defmodule Astarte.Housekeeping.ReleaseTasks do
   end
 
   defp wait_connection_and_check_astarte_keyspace(retries \\ 60) do
-    case Queries.is_astarte_keyspace_existing() do
+    case Keyspace.keyspace_existing?("astarte") do
```

Comment:
could be shortened as 

```suggestion
    with {:error, :database_connection_error} <- Keyspace.keyspace_existing?("astarte") do
      if retries > 0 do
          :timer.sleep(1000)
          wait_connection_and_check_astarte_keyspace(retries - 1)
        else
          {:error, :database_connection_error}
        end
    
    end
```

cause when 'with' can not match, it returns the value (so {:ok, _} or {:error, _})